### ISSUE TITLE
[Accessibility] Voiceover convey the incorrect position of the 'Snooze For' dropdown menu items under the 'CalendarReminder JSON' screen.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6284]
### Target Platforms

iOS

### SDK Version

App Version: Version 1.0 (2.3.1-beta.20210603.1)

### Application Name

Visualizer

### Problem Description

[33751792](https://microsoft.visualstudio.com/OS/_workitems/edit/33751792)

Voiceover should convey the correct correct position of the menu items. It should announce as '5 minutes 1 of 3 , 15 minutes 2 of 3'

If voiceover is not reading the position of list items in “Snooze For” dropdown on “CalendarReminder” page then voiceover user will not be able to find out the position of list items and will be confused and face difficulties to navigate in the list on screen.

### Card JSON

```JSON
https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.0/Scenarios/CalendarReminder.json
```

**Comment by vagpt** [comment]
Not repro on the latest build hence closing this issue.
